<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pBtn_start_stop">
        <property name="text">
         <string>Старт</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pBtn_loop">
        <property name="text">
         <string>Круг</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pBtn_clear">
        <property name="text">
         <string>Очистить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pBtn_exit">
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QTextBrowser" name="txtBr_loopData"/>
    </item>
    <item row="0" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lb_time_headline">
        <property name="text">
         <string>Время:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_time">
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>280</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pBtn_exit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
